frontend/src/components/Video.jsx

```diff
@@ -54,6 +55,9 @@ function Video(props) {
       >
         <track kind="captions" />
       </video>
+      <Typography variant="overline" display="block" gutterBottom>
+        {name}
+      </Typography>
```

Comment:
I'd probably add something like this, just so we don't create an empty element if there is no name:
```suggestion
      {!!name && (
        <Typography variant="overline" display="block" gutterBottom>
          {name}
        </Typography>
      })
```